easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2022b-AmberTools-22.3.eb

```diff
@@ -0,0 +1,96 @@
+name = 'Amber'
+local_amber_ver = 22
+local_ambertools_ver = 22
+# Patch levels from https://ambermd.org/AmberPatches.php and https://ambermd.org/ATPatches.php
+patchlevels = (3, 0)  # (AmberTools, Amber)
```

Comment:
Please update this to include the latest patches, so `patchlevels = (5, 3)` ([AmberTools](https://ambermd.org/ATPatches.php), [Amber](https://ambermd.org/AmberPatches.php))

This means that the filename should be altered accordingly, to be `Amber-22.3-foss-2022b-AmberTools-22.5.eb`

Reply:
This software tooks me days to run on our cluster.
Happy to got it work with 2022b stuff. Can't test it with AmberTools 22.5 now either can change the productive work environment for our chemican staff now.

Maybe we just drop the PR or AmberTools 22.5 variant may be tested elsewhere.